### ISSUE TITLE
TS Norwich doesn't have a 'teacher-info' page

### ISSUE THREAD

**Issue by laurabillings** [orzubalsky/tradeschool#129]
![screen shot 2014-02-19 at 09 34 34](https://f.cloud.github.com/assets/6553938/2205086/1fd20d66-9949-11e3-9d02-9a353608c78f.png)
Norwich are starting a new season,and have added dates on the backend. But there isn't a 'register to teach / teacher info' page showing up on the list of pages. (And I can't create hyperlinks in the text on the homepage to click through to it either)

**Comment by orzubalsky** [comment]
You can add a teacher info page by adding a new branch page on the admin at http://tradeschool.coop/admin/tradeschool/page/

I see that you linked to /norwich/page/teacher-info/, so make sure that the URL field in the form is teacher-info. 

The HTML we usually use is:

```
<h3>Here's a short&nbsp;orientation to get you situated.</h3>
<h4><strong>How Trade School works</strong></h4>
<p>Trade School is a non-traditional learning space where students barter with teachers.</p>
 <p>After you submit your class idea, when you'd like to teach, and what you want in exchange for    teaching, Trade School organizers will review your submission. If your submission is approved, it will be posted on the website for students to sign up. On the day of class, you will meet your students at Trade School, receive your barter items, and enjoy the class.</p>
<h4><strong>Your class</strong></h4>
<p>Your class can be about anything and can be structured in any way you like. Just be sure that you feel comfortable setting the tone and sharing information. You are in charge during your class! Past topics and approaches to teaching have ranged from butter-making workshops to foraging walking tours; from business lectures to group discussions. We find that students want to get to know each other, so classes work best with break-out groups, work in pairs, and lots of student participation.</p>
<h4><strong>Your barter items</strong></h4>
<p>Think of the barter items you ask for as a giant wish list. Make it as long and as creative as you can. Last year, someone asked for new size 9.5 women's running shoes, and got them. This is a way for students to get to know you, or to begin learning about your topic before class begins. Try to ask for skills and non-monetary things (like research, advice, tips, help cleaning up, enthusiasm) as much as monetary-requests (like clothes, food, materials, etc.).</p>
<p>&nbsp;</p>
<div class="subButtonBox"><a class="submit" href="../class/add">Sign up to teach</a></div>
```

I went ahead and created one for Norwich.

**Comment by laurabillings** [comment]
![screen shot 2014-02-21 at 10 09 46](https://f.cloud.github.com/assets/6553938/2228708/57679032-9ae0-11e3-886d-23cf30cc94ca.png)
Hey Or
When I try to add HTML I get a blank box - the same as trying to add links.
Can you please make a teacher page for Rotterdam so I can help Nieneke get
set up? I can't add this page to her branch.

On Wed, Feb 19, 2014 at 2:56 PM, or zubalsky notifications@github.comwrote:

> Closed #129 https://github.com/orzubalsky/tradeschool/issues/129.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/orzubalsky/tradeschool/issues/129
> .

## 

Laura Billings

07789 214 934
@laurabillings https://twitter.com/laurabillings

**Comment by laurabillings** [comment]
Ooh - wait . It's working now! YEEEAHHH!!
